Sources/Datadog/Logger.swift

```diff
@@ -58,7 +58,8 @@ public enum LogLevel: Int, Codable {
 public typealias DDLogger = Logger
 
 public class Logger {
-    internal typealias LogEventValidation = (LogEvent) -> Bool
+    /// Filters logs by their status.
+    internal typealias LogsFilter = (LogEvent.Status) -> Bool
```

Comment:
Renamed to "logs filter" to better express its purpose. Anyway, this API will change (disappear) very soon as part of our remaining enhancement after internal loggers refactoring.